Readme.txt

```diff
@@ -1,41 +1,34 @@
-                              MORSE RUNNER  1.71
+                                MORSE RUNNER
                               Contest Simulator
-
                                   freeware
+                                  
+                          Field Day Contest Prototype
 
-               Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA
-
-                      http://www.dxatlas.com/MorseRunner/
-
```

Comment:
According to https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects, we should not change the original copyright notice of an existing author.

This article also recommends a more basic approach to Copyright messages using a general form of:
        `Copyright (C) Contributors to the Morse Runner Contest Simulator Project`

Other links regarding copyright notices:
- https://opensource.com/article/20/10/copyright-notices-open-source-software

So, I'm inclined to simply add a generic statement, something like `Copyright (C) Contributors to Morse Runner Project` below Alex's original copyright message. I would like Alex's URL below his copyright. Perhaps after his, we can include the new one along with the path to the community edition.

Perhaps something like:
```
               Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA
                      http://www.dxatlas.com/MorseRunner/

                Copyright (C) 2016-2022 Morse Runner Project Contributors
```

I notice at the bottom of our [LICENCING.md](https://github.com/w7sst/MorseRunner/blob/main/LICENSE.md) file, In APPENDIX A, it says `You may add additional accurate notices of copyright ownership.` This certainly implies that we can add additional copyright notices. I tend to follow the recommendation of the article above recommending adding the general `Copyright (C) Morse Runner Project Contributors` or `Copyright (C) Contributors to the Morse Runner Project`. I don't have a strong feeling on this. However, I am not sure I want my name listed (I'm fine with my callsign).

Reply:
I didn't change the existing copyright for Alex. I listed the scope of his copyright (the versions he authored). I listed these in descending order because I believe it is less confusing for the reader. They downloaded your fork - your version - of this software, so when they open the readme they should see your version number first to know they have the right file. Otherwise they download 1.80 and immediately see 1.68 in the readme and think old file. They should see your work first because that is what they are using. Listing the author's you forked and their respective copyright is necessary such as Lin Quan's. You are building on top of their work and therefor are listed on top. 

I'm not a lawyer, but I do know legal matters are no place for generalities or ambiguity. I'll gladly add the generic copyright but think it is only fair we list the individuals underneath as well. Not everyone can code Delphi.   

We do identify with our callsigns but they are our stations, not our persons. Our callsign can not own property. Besides W7SST was Orville Longfellow of 1200 Canyon, Ogden Utah in 1947. Your call and your name with date is precise. You should be as proud of your work as I am. I'm just a cheerleader but you are doing it. This is good work on Alex's program.

Reply:
So I really should list Alex's copyright last according to my reasoning above. But I noticed with the previous versions I started with Alex. I need to add a space before previous works so that this can be understood to be in a chronological order and different from the current copyright notice. 

I didn't make any changes for now. I wanted to see what you guys think of my reasoning, and get David's take on this as well.

Reply:
Should we keep the version number in line 1 of this Readme.txt file?

Reply:
I did read in one of the links that I posted above that an existing copyright message should not be modified. So I'm wondering if we should be adding the version numbers in front of existing copyright messages.

I also read that with open source software, it is assumed that your work is open and will be modified by others. I'm just thinking of what this section looks like in 5 years, there will be lots of copyright lines and it will be hard to maintain. I do notice some open source projects will maintain a list of contributors in a separate file - perhaps we should look into this instead of list everyone here. This is another reason I'm recommending the generic Copyright message reference all contributors to this project.

As we integrate other branches from other authors, they may include a copyright that we will need to move forward. I did a quick grep in the versions I have on my laptop, and I did not see any additional copyright messages added. However, I did notice that F6FVY added some "modified statements" after Alex's copyright...
```
               Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA

                           Modified by N2IC, 2016.
                           Modified by F6FVY, 2018.

                      http://www.dxatlas.com/MorseRunner/
```

When we merge in F6FVY's code, these comments do not need to be preserved as they have been captured in the version information section that you are adding and maintaining.

Reply:
Hi all,

I'm not a lawyer either, but after several readings it seems that it's not that easy to establish copyright over modifications. 

The list of contributors could be a good way.

Reply:
I agree we can maintain a list of contributors in the repository.

To keep this simple, how about this for now?

```
               Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA
                      http://www.dxatlas.com/MorseRunner/

          Copyright (C) Morse Runner Community Edition Contributors
                      https://www.github.com/w7sst/MorseRunner/
```
After we ask the other prior authors to join the project, we can revisit this discussion. Again, my concern is that this section will become messy after 5-10 years of contributions. All future contributions for this project will originate from the community, so adding the copyright message for the community is sufficient. Keep in mind that no other prior authors have added a copyright message to this readme file.

So I'm wondering if maintaining a list of contributors to the repository is sufficient for giving developer credit.

Also, developers will receive proper credit in the Version Information were we will be adding "(Coded by <developer>)." This convention is used by N1MM and I think we are adopting it here as well.

Reply:
I modified Readme.txt again with the JOIN OUR COMMUNITY section and a proposed copyright string. It is best to view this in the full file (use the view file command in the "..." pick above.

@scotthibbs , I'll let you chime in here again. I'm leaning towards the simpler format. We can also revisit this question with the other authors once they join the project. For now, since none of them had explicitly added their own copyright to this file, I'm leaning to not add that copyright now.

@ct7aup, Thank you for contributing your thoughts to this discussion. We appreciate it.

Reply:
@scotthibbs , we still need to resolve the final copyright at the top of this file. The file now contains two options and we need to pick one and make a final edit to this file. I am recommending we use the simpler version.

You need to view the file as a whole in order to see the two options as it gets lost in this review format. Let me know if you want me to make the final edit.

Thoughts?